scripts/utils.ts

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env npx tsx
+
+import { EC2 } from "@aws-sdk/client-ec2";
+
+export interface EC2Instance {
+  InstanceId?: string;
+  State?: { Name?: string };
+  Tags?: Array<{ Key?: string; Value?: string }>;
+}
+
+export class EC2Utils {
```

Comment:
Instead of a `EC2Utils` class, let's make all of these private methods as exportable methods from this file and take in the `ec2: EC2` as an argument